src/main/java/com/comphenix/protocol/utility/MinecraftVersion.java

```diff
@@ -40,7 +40,7 @@ public final class MinecraftVersion implements Comparable<MinecraftVersion>, Ser
 	/**
 	 * Version 1.19.3 - introducing feature preview
 	 */
-	public static final MinecraftVersion FEATURE_PREVIEW_UPDATE = new MinecraftVersion("1.19.3");
+	public static final MinecraftVersion FEATURE_PREVIEW_UPDATE = new MinecraftVersion("1.19.4");
```

Comment:
Not sure if you should change this one, 1.19.3 introduced some new features related to packets and people might use this constant to check if they should enable them

Reply:
For example here: 
https://github.com/juliarn/NPC-Lib/blob/db1a4fd803c09b31c2c46cc92720f572d85d84ef/bukkit/src/main/java/com/github/juliarn/npclib/bukkit/protocol/ProtocolLibPacketAdapter.java#L304-L315

Reply:
Yes, I realized it later and have it reverted locally... I am now still thinking about how to rework the WrappedServerPing, because as a record it is immutable while the old one was mutable... I am a bit afraid that it will be a breaking change...